irods_testing_environment/irods_setup.py

```diff
@@ -380,25 +382,25 @@ def build_unattended_install_input_for_catalog_consumer(self):
         # common for tests to modify server_config.json to meet their needs.
         json_input = {
             "admin_password": self.admin_password,
-            "default_resource_directory": '/var/lib/irods/Vault', #self.vault_directory,
+            "default_resource_directory": '/var/lib/irods/Vault',
```

Comment:
Try running topology-from-consumer tests. Just want to make sure it's not stomping any assumptions